src/sprite/sprite.hpp

```diff
@@ -124,8 +124,7 @@ class Sprite final
   Blend get_blend() const;
 
   bool has_action (const std::string& name) const { return (m_data.get_action(name) != nullptr); }
-
-  std::size_t get_actions_count() { return m_data.actions.size(); }
+  size_t get_actions_count() const { return m_data.actions.size(); }
```

Comment:
what

Reply:
Nowhere does the code specify `std::` for `size_t`, since there is no need to. It's a nitpick but I was editing the code anyways, so...